manual/src/encryption.md

```diff
@@ -0,0 +1,774 @@
+# Unstable API
+
+To enable support for in-use encryption (client-side field level encryption and queryable encryption), enable the `"in-use-encryption-unstable"` feature of the `mongodb` crate.  As the name implies, the API for this feature is unstable, and may change in backwards-incompatible ways in minor releases.
+
+# Client-Side Field Level Encryption
+
+Starting with MongoDB 4.2, client-side field level encryption allows an application to encrypt specific data fields in addition to pre-existing MongoDB encryption features such as [Encryption at Rest](https://dochub.mongodb.org/core/security-encryption-at-rest) and [TLS/SSL (Transport Encryption)](https://dochub.mongodb.org/core/security-tls-transport-encryption).
+
+With field level encryption, applications can encrypt fields in documents prior to transmitting data over the wire to the server. Client-side field level encryption supports workloads where applications must guarantee that unauthorized parties, including server administrators, cannot read the encrypted data.
+
+See also the MongoDB documentation on [Client Side Field Level Encryption](https://dochub.mongodb.org/core/client-side-field-level-encryption).
+
+## Dependencies
+
+To get started using client-side field level encryption in your project, you will need to install [libmongocrypt](https://github.com/mongodb/libmongocrypt), which can be compiled from source or fetched from a variety of package repositories; for more information, see the libmongocrypt [README](https://github.com/mongodb/libmongocrypt/blob/master/README.md).  If you install libmongocrypt in a location outside of the system library search path, the `MONGOCRYPT_LIB_DIR` environment variable will need to be set when compiling your project.
```

Comment:
the docs version of install instructions might be a little friendlier than the README, since that has a lot of info specific to driver developers? https://www.mongodb.com/docs/manual/core/csfle/reference/libmongocrypt/#std-label-csfle-reference-libmongocrypt

Reply:
Ah, very nice, updated.  Also removed the reference to compiling from source given the warning on that page.